package.json

```diff
@@ -179,5 +187,6 @@
   ],
   "publishConfig": {
     "access": "public"
-  }
+  },
+  "types": "./typings/index.d.ts"
```

Comment:
This small change will break type interference as during build in CI only dist directory is persisted to the workspace.